configs/stage/roles/provisioning.json

```diff
@@ -1,16 +1,18 @@
 {
     "roles": [{
         "name": "Launch Administrator",
-        "description": "A launch administrator role that grants read and write permissions.",
+        "display_name": "Launch administrator",
+        "description": "Perform read and update operations on launch reservation and related resources.",
         "system": true,
         "admin_default": true,
-        "version": 2,
+        "version": 3,
         "access": [{
             "permission": "provisioning:*:*"
         }]
     }, {
         "name": "Launch Viewer",
-        "description": "An launch role that grants read permissions on launch reservation and related resources.",
+        "display_name": "Launch viewer",
+        "description": "Perform read operations on launch reservation and related resources.",
```

Comment:
please for this change is needed to increase the version

Reply:
"version": 2,   
change to
"version": 3